learn/faq.md

```diff
@@ -64,6 +64,29 @@ A [dependency graph](https://en.wikipedia.org/wiki/Dependency_graph) is a tree-l
 
 Each separate conda environment would have its own dependency graph. The items in a dependency graph would be the packages that conda manages and what each of those packages require as a prerequisite to function properly.
 
+
+## Why can't conda find this package that I know exists? 
+
+There are several reasons why conda might be unable to find a package that you know exists. Here are some troubleshooting steps to try:
+
+* Check the channel: One common reason is that the package is not available in the channel that you are searching in. It's possible that the package was removed from the channel, or that the channel itself is not properly configured.
+
+* Update your index: Another possible reason is that you have an outdated or incomplete index of available packages. You can update your index by running `conda update --all` or `conda index`. It's also possible that the package you are searching for is not compatible with your operating system or architecture.
+
+* Check your conda installation: It's possible that there is a problem with your conda installation or configuration. You can try reinstalling conda or creating a new environment to see if that resolves the issue.
+
+* Check a different channel: Sometimes a package may be available in a different channel than the one you are using. You can try searching for the package in different channels using the conda search command. For example, `conda search -c conda-forge my_package`.
+
+* Update conda: It's possible that the version of conda you are using is outdated and can't find the package. You can try updating conda using the command `conda update conda`.
```

Comment:
General question (perhaps for @travishathaway or @kenodegard): A lot of people do `conda update conda` in (base). Is that advised? If not, should probably add an advisement against that. Just checking.